api-server/scanner-lib/src/blockchain_state/adapter.rs

```diff
@@ -83,8 +77,8 @@ impl PoSAccountingView for PoSAdapter {
         Ok(self.pools.get(&pool_id).cloned())
     }
 
-    fn get_pool_balance(&self, pool_id: PoolId) -> Result<Option<Amount>, Self::Error> {
-        Ok(self.pools.get(&pool_id).map(|data| data.pledge_amount()))
+    fn get_pool_balance(&self, _pool_id: PoolId) -> Result<Option<Amount>, Self::Error> {
+        unimplemented!()
```

Comment:
Why?

Reply:
The implementation was incorrect. Pool balance is pledge + all delegation which is no accounted here. But since this function is never used I just removed the implementation